Source/WTF/wtf/text/AtomStringImpl.cpp

```diff
@@ -281,12 +281,12 @@ struct BufferFromStaticDataTranslator {
         return buf.hash;
     }
 
-    static bool equal(PackedPtr<StringImpl> const& str, const Buffer& buf)
+    static bool equal(typename AtomStringTable::StringEntry const& str, const Buffer& buf)
     {
         return WTF::equal(str.get(), buf.characters, buf.length);
     }
 
-    static void translate(PackedPtr<StringImpl>& location, const Buffer& buf, unsigned hash)
+    static void translate(typename AtomStringTable::StringEntry& location, const Buffer& buf, unsigned hash)
```

Comment:
Ditto.

Reply:
It seems that this is not necessary. Dropped. (I originally though it is necessary because this class is templatized).